chia/pools/pool_wallet.py

```diff
@@ -1001,4 +1004,4 @@ def puzzle_hash_for_pk(self, pubkey: G1Element) -> bytes32:
 if TYPE_CHECKING:
     from chia.wallet.wallet_protocol import WalletProtocol
 
-    _dummy: WalletProtocol = PoolWallet()
+    _dummy: WalletProtocol = PoolWallet()  # type: ignore[call-arg]  # Ignore missing args for type checks
```

Comment:
There are other options for this check.  I'll follow up on this soon and adjust them all.  For now this is an easy change.

```suggestion
    _dummy: WalletProtocol = cast(PoolWallet, None)
```